app/browser/api/userModel.js

```diff
@@ -0,0 +1,752 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+const um = require('@brave-intl/bat-usermodel')
+const elph = require('@brave-intl/bat-elph')
+const braveNotifier = require('brave-node-notifier')
+const path = require('path')
+const getSSID = require('detect-ssid')
+const underscore = require('underscore')
+const url = require('url')
```

Comment:
replace this with `const urlFormat = require('url').format` so that url.parse is not callable (which is insecure and should be always replaced with our urlParse module)

Reply:
ok.